mattermost/mattermost-webapp
# [MM-28188] e2e cypress tests for subpath

#### Summary

- Tests for MM-T985, MM-T986, MM-T987, MM-T988, MM-T989
- Add docs for how to run subpath tests
- Add configuration to `cypress.json` for specifying other subpath server
- Add control flag to check if test should run under subpath. 
 

#### Ticket Link

- https://mattermost.atlassian.net/browse/MM-28188
